napari/utils/key_bindings.py

```diff
@@ -60,6 +60,8 @@ def hello_world(viewer):
 USER_KEYMAP: Mapping[str, Callable] = {}
 
 KEY_SUBS = {
+    'Command': 'Meta',
```

Comment:
I don't understand how this interacts with the 0.5.0 -> 0.6.0 migrator.

But this addition seems reasonable, assuming we want cross-platform consistency.

Reply:
Oh sorry this line specifically is ancillary, I'm not even sure Command is a valid shortcut currently, *but* I wanted to be able to write Command in a config file and have it work 😂 Sorry that is a red herring in review!

Reply:
An explicit test would probably be enough for me to understand though.